_transactions.md

```diff
@@ -118,15 +114,15 @@ In step one, one prepares the transaction by specifying:
 - The _currency_ to denominate the transaction by.
 - The _amount_ of value to send in the denominated currency.
 - The _origin_ of the transaction can be an account id in the case of a _deposit_.
-- The _destination_ of the transaction, which can be in the form of a bitcoin address, an email address, an account id, an application id or an Uphold username.
+- The _destination_ of the transaction, which can be in the form of a bitcoin address, an email address, an account id, an application id or an Uphold user email.
```

Comment:
This one should probably have been removed [as well](https://github.com/uphold/docs/pull/175#discussion_r339991478), right @rplopes?

Reply:
😱 

You're right, that's repeated as well.

Reply:
Thanks for confirming. Submitted #176 to fix.